test/__tests__/data/algolia_router_output.json

```diff
@@ -6,10 +6,8 @@
       "statusCode": 400,
       "error": "Missing required value from \"properties.index\"",
       "statTags": {
```

Comment:
`destType` is supposed to be included at the core level. Can you check what's going on?